backend/src/handlers/grade.ts

```diff
@@ -159,30 +196,28 @@ export const getQuestionsGrades = async (
 					},
 				});
 
+				let studentTotal = 0;
+
+				user?.get().QuestionGrades?.forEach((grade) => {
+					studentTotal += grade.points;
+				});
+
+				classTotal += studentTotal;
+
 				return {
 					name: student.name,
+					total: studentTotal,
 					...user?.get(),
 				};
 			})
 		);
 
-		// Get all questions belong to the current session
-		const session = await Session.findOne({
-			where: {
-				id: sessionId,
-			},
-			include: {
-				model: Question,
-				attributes: ['id', 'title'],
-			},
-		});
-		if (!session) {
-			return responses.notFound();
-		}
+		const classAverage = classTotal / canvasStudents.length;
 
 		return responses.ok({
+			classAverage: { points: classAverage, maxPoints: classMax },
```

Comment:
whoa whoa why are we calculating the class max? comparing yourself to others is incredibly detrimental to a person's self-esteem.